index.js

```diff
@@ -67,6 +67,45 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(){
+    console.log('title clicked')
+
+    toggleColor(header)
+
+} )
+
```

Comment:
nicely done with this deliverable and happy to see the use of console log for debugging! 
Using arrow functions (ES6 feature), you could also write this solution as shown below. This is the same exact functionality!
`header.addEventListener("click", () => toggleColor(header))`